### ISSUE TITLE
Consistent Compilation Issue : TypeError [ERR_INVALID_REPL_INPUT]

### ISSUE THREAD

**Issue by bijoy26** [preeti13456/Blockchain-project#11]
### Upon clear compilation and deployment understanding, I once again went through the entire project structure and after doing some adjustments, I tried to compile the contract files several times.
During compilation. I received consistent **"TypeError [ERR_INVALID_REPL_INPUT]: Listeners for `uncaughtException` cannot be used in the REPL"**, which didn't seem to fix up even after trying with several *solc* compiler and *truffle* versions.
 
### **To Reproduce**
1. First create a separate branch to simulate this out
2. Move the **"contracts/Migrations/deploy_cotracts.js"** file into "migration/" folder (truffle requirement) and rename it as **"2_deploy_cotracts.js"** (truffle requirement)
3. Go into **contracts/election.sol** and comment out following lines at the bottom (redeclaration of same contract)
```
// contract Election {
//  ...
//  event votedEvent (
//    uint indexed _candidateId
//  );
```
4. Now launch **Truffle Develop** local server using`$ truffle develop` and then use `compile`. You will have the following error.

![1](https://user-images.githubusercontent.com/42487891/87433465-27080b80-c60b-11ea-9775-cd3468e0b253.png)

Fix compiler version on the **truffle-config.js** file by adding following snippet on the bottom -
```
  compilers: {
    solc: {
      version: "0.4.2"    // Fetch exact version from solc-bin (default: truffle's version)
      // docker: true,        // Use "0.5.1" you've installed locally with docker (default: false)
      // settings: {          // See the solidity docs for advice about optimization and evmVersion
      //  optimizer: {
      //    enabled: false,
      //    runs: 200
      //  },
      //  evmVersion: "byzantium"
      // }
    }
  }
```

5. Now try compiling again and you will likely to receive following error -
`TypeError [ERR_INVALID_REPL_INPUT]: Listeners for 'uncaughtException' cannot be used in the REPL `

![2](https://user-images.githubusercontent.com/42487891/87432502-e8258600-c609-11ea-8803-d7ae69dfb20b.png)

###  **Desktop Specs:**
 - OS: Windows 10 64 bit
 - RAM: 8 GB 
 - CPU: Core i5 5440

### **Additional context**
 There are not much solutions over internet but [here is the related issue](https://github.com/trufflesuite/truffle/issues/2070) on *Truffle Github* where this behavior is explained in detail.
Some of them suggested - 
> easiest way out of this issue is to install node v11.15.0. As node v12 and v13 both will encounter the issue

I guess this error is version dependent so @preeti13456 and @manasir66  you both should check and test this and then we can come to an agreement on the solution.

**Comment by bijoy26** [comment]
Issue addressed and fixed at #13 .
@preeti13456 have a thorough inspection.

**Comment by preeti13456** [comment]
Good work
